com.unity.render-pipelines.universal/ShaderLibrary/Debug/DebuggingFullscreen.hlsl

```diff
@@ -3,25 +3,21 @@
 
 #if defined(DEBUG_DISPLAY)
 
+int _ValidationChannels;
 float _RangeMinimum;
 float _RangeMaximum;
-int _HighlightOutOfRangeAlpha;
 
-TEXTURE2D(_DebugTexture);        SAMPLER(sampler_DebugTexture);
+TEXTURE2D_X(_DebugTexture);
+TEXTURE2D(_DebugTextureNoStereo);
```

Comment:
You don't need to declare two textures. TEXTURE2D_X should work on both stereo and non-stereo. Then when sampling, the _X macro should work as well in both cases.

Reply:
Normally that's true, but in this case I need "NoStereo" variant for textures that are Texture2D, even when XR is on. This is the case for shadow maps -  they are not texture arrays even in XR.

Reply:
Ah I see.